index.js

```diff
@@ -1,22 +1,29 @@
 /*
-	MIT License http://www.opensource.org/licenses/mit-license.php
-	Author Tobias Koppers @sokra
-*/
+ MIT License http://www.opensource.org/licenses/mit-license.php
+ Author Tobias Koppers @sokra
+ */
 var loaderUtils = require("loader-utils");
 var mime = require("mime");
+
 module.exports = function(content) {
-	this.cacheable && this.cacheable();
```

Comment:
Can we really drop it yet ? I'm not against it, but `wepack =< v1.0.0` users will lose caching then.